src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -0,0 +1,133 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from datetime import datetime
+import json
+import logging
+import os
+from os import listdir
+from os.path import isfile, join
+import sas
+import sas_testcase
+
+from util import winnforum_testcase, configurable_testcase, writeConfig, loadConfig, computePropagationAntennaModel
+  
+def cbsddata(device):
+    installationParam = device['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = None
+    if 'antennaBeamwidth' not in installationParam:
+        installationParam['antennaBeamwidth'] = None
+    if 'antennaGain' not in installationParam:
+        installationParam['antennaGain'] = 0
+                
+    cbsd = {'latitude': installationParam['latitude'],
+             'longitude': installationParam['longitude'],
+              'height': installationParam['height'],
+              'heightType': installationParam['heightType'],
+              'indoorDeployment': installationParam['indoorDeployment'],
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaBeamwidth': installationParam['antennaBeamwidth'],
+              'cbsdCategory': device['cbsdCategory']}
+    return cbsd
+
+def fssdata(fss_record, rx_antenna_gain_required=None):
+    deploymentParam = fss_record['deploymentParam'][0]
+    installationParam = deploymentParam['installationParam']
+    if 'antennaAzimuth' not in installationParam:
+        installationParam['antennaAzimuth'] = None
+        
+    fss = {'latitude': installationParam['latitude'],
+             'longitude': installationParam['longitude'],
+              'height': installationParam['height'],
+              'antennaAzimuth': installationParam['antennaAzimuth'],
+              'antennaGain': installationParam['antennaGain'],
+              'antennaElevation': -installationParam['antennaDowntilt']}
```

Comment:
Correct. 
Kate mentioned earlier that the name of FSS in installationParam would be 'antennaElevation', but this is probably incorrect. Although this is what is defined in the **SAS-under-test testing API** document, the FSS json reference record uses antennaDowntilt, and it is probably better to continue using that so there is no mismatch with the CBSD installationParams (all info required for FSS antenna processing is present in that installationParams).

So in conclusion: this is ok for me.